index.js

```diff
@@ -142,10 +143,18 @@ function buildRouteRate (pluginComponent, params, routeOptions) {
         }
 
         res.code(429)
-          .header('X-RateLimit-Limit', params.max)
```

Comment:
Use the kebab-case notation https://developer.twitter.com/en/docs/basics/rate-limiting it's more common